concrete/src/Cache/Page/PageCache.php

```diff
@@ -19,7 +19,7 @@
     /**
      * @var \Concrete\Core\Cache\Page\PageCache|null
      */
-    static $library;
+    protected static $library;
```

Comment:
This might be a breaking change for some, worth saving this for v9?

Reply:
As I wrote in the commit https://github.com/concrete5/concrete5/pull/7133/commits/19fde37de8aa373fe0aa8b94151860ee215ba51f, if someone is accessing `$library` from outside this class (or its sub-classes), is doing something wrong because this variable can be uninitialized. BTW, yes, we can mark it as `public` and `@deprecated`.

Reply:
Done: 61cb7c56a3e2f022605d152d6a8d2fc5901c6a1f